spec/units/view_collection_spec.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+
+describe 'ViewCollection' do
+  let(:view_collection) { Blueprinter::ViewCollection.new }
+
+  let!(:default_view) { view_collection[:default] }
+  let!(:view) { view_collection[:view] }
```

Comment:
Why are we using `let!` here?

Reply:
When a view collection tries to fetch a view that doesn't exist it creates it, which is what we're doing here. We need that view to exist for our tests, but don't always call `view` directly, so it won't be initialized unless we use `let!`.